### PR TITLE
Api diff between 2.2 and 3.0 netcoreapp

### PR DESCRIPTION
**Pull request by Anipik**
cc @ahsonkhan @danmosemsft @terrajobst @ericstj @karelz @buyaa-n, @krwq @layomia, @JeremyKuhne, @safern @divega, @ajcvickers,@cheenamalhotra, @david-engel, @Gary-Zh @wtgodbe, @maryamariyan @carlossanlop @davidsh, @wfurt, @scalablecory, @eiriktsarpalis, @karelz
@tannergooding, @steveharter, @GrabYourPitchforks 
@richlander 

guys please verify the changes in your area owners.

### CODE REVIEW

**Comment by danmosemsft** [comment]
We did a lot of work 🥂

**Comment by ericstj** [comment]
@cstoner for VB

**Comment by cstoner** [comment]
> @cstoner for VB

I think you @-ed the wrong person.

**Comment by danmosemsft** [comment]
@cston is the correct person (thanks @cstoner)

**Comment by tannergooding** [comment]
`System.Numerics` (including the tangential ones like `System.Math`) and `System.Runtime.Intrinsics.*` look good to me.

**Comment by cston** [comment]
`Microsoft.VisualBasic.*` look good to me.

File: release-notes/3.0/preview/api-diff/3.0 vs 2.2/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# .NET Core 3.0 API Changes
+
+The following API changes were made in .NET Core 3.0:
+
+- [.NET Core](./3.0.0.md)
+- [.NET Core NuGet Packages](./standalone-packages/3.0.0-standalone-packages.md)
```
Comment (ahsonkhan) [inline release-notes/3.0/preview/api-diff/3.0 vs 2.2/README.md]: Do we want to add aspnet?

**Comment by David-Engel** [comment]
`System.Data.SqlClient` Looks good to me.

**Comment by ahsonkhan** [comment]
System.Buffers, System.Text.Json/Serialization, System, looks good.

File: release-notes/3.0/preview/api-diff/3.0 vs 2.2/standalone-packages/3.0.0-standalone-packages_System.Runtime.CompilerServices.Unsafe.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+# System.Runtime.CompilerServices.Unsafe
+
+``` diff
+ {
+ {
+-    namespace System.Runtime.CompilerServices {
```
Comment (ahsonkhan) [inline release-notes/3.0/preview/api-diff/3.0 vs 2.2/standalone-packages/3.0.0-standalone-packages_System.Runtime.CompilerServices.Unsafe.md]: Did we move `System.Runtime.CompilerServices.Unsafe`in-box? I wasn't aware of this. Is this expected? We still ship the OOB package, but I thought this was a "pure" OOB:
https://github.com/dotnet/corefx/blob/b893e604ce832ccf4e9cbcbdfe0b0ae2b23195bf/src/System.Runtime.CompilerServices.Unsafe/pkg/System.Runtime.CompilerServices.Unsafe.pkgproj#L19

https://github.com/dotnet/corefx/pull/25959

https://github.com/dotnet/corefx/issues/26145#issuecomment-355679947
https://github.com/dotnet/corefx/pull/28395

cc @ericstj, @jkotas

File: release-notes/3.0/preview/api-diff/3.0 vs 2.2/README.md (hunk 1 above)
Comment (Anipik) [reply release-notes/3.0/preview/api-diff/3.0 vs 2.2/README.md]: They didn't publish any ref assemblies for 2.1 and 2.2. So I am trying to find a way to get the diff but it seems unlikely

Comment (ahsonkhan) [reply release-notes/3.0/preview/api-diff/3.0 vs 2.2/README.md]: Sounds good to me. Thanks for checking.

File: release-notes/3.0/preview/api-diff/3.0 vs 2.2/standalone-packages/3.0.0-standalone-packages_System.Runtime.CompilerServices.Unsafe.md (hunk 2 above)
Comment (ericstj) [reply release-notes/3.0/preview/api-diff/3.0 vs 2.2/standalone-packages/3.0.0-standalone-packages_System.Runtime.CompilerServices.Unsafe.md]: We build the same binary to ship both inbox and OOB.  There still remains an `Internal.Runtime.CompilerServices.Unsafe` type in System.Private.CoreLib but we don't unify this type `System.Runtime.CompilerServices.Unsafe`.

Here is the record of this decision: https://github.com/dotnet/corefx/issues/33508

Comment (ericstj) [reply release-notes/3.0/preview/api-diff/3.0 vs 2.2/standalone-packages/3.0.0-standalone-packages_System.Runtime.CompilerServices.Unsafe.md]: @Anipik I don't think it's correct to represent this as a diff.  Something is wonky with your method for producing the .md

**Pull request merged**